Sources/Templates/AutoMockable.stencil

```diff
@@ -204,6 +207,9 @@ class ExampleTest: XCTestCase {
 {% endmacro %}
 
 {% macro mockVariable variable %}
+{% if variable|annotated:"IfCanImport" %}
+    #if canImport({{ variable.annotations["IfCanImport"] }})
+{% endif %}
```

Comment:
This is not new logic, we already do this for mocked functions. This PR makes it so we also do this logic for mocked properties.